src/ssec/giirs_ssec2ioda.py

```diff
@@ -98,40 +110,57 @@ def readInFiles(self, filenames):
         qcKey = self.VAR_NAME, self.writer.OqcName()
 
         numFiles = 0
+        totalFiles = 0
         self.writer._nlocs = 0
 
         sat_zen_max = 74
         detector_number = 128
-        detector_array = np.arange(1, detector_number+1, dtype='i4')
+        detector_array = np.arange(1, detector_number+1, dtype=np.float32)
```

Comment:
Why the type change from int to float?  It looks like this is used to create the scan_position@MetaData variable, which I would have expected to be an integer. @weihanucar, is scan_position@MetaData supposed to be an integer or a float?

Reply:
Yes, thanks for catching that.  Is addressed with new changes.

Reply:
It turns out that this change leads to a runtime failure in ufo/crtm.  See [ufo # 1285](https://github.com/JCSDA/ufo/issues/1285).